auth/auth_openshift.go

```diff
@@ -83,10 +86,11 @@ func newOpenShiftAuth(ctx context.Context, c *openShiftConfig) (oauth2.Endpoint,
 		return oauth2.Endpoint{}, nil, err
 	}
 
+	kubeAdminLogoutURL := proxy.SingleJoiningSlash(metadata.Issuer, "/logout")
```

Comment:
kind of curious why we have this func vs just using `filepath.Join()` but I'm sure thats history.

Reply:
I think it might normalize `//` to `/` in the URL scheme

Reply:
```
package main

import (
	"fmt"
	"path/filepath"
)

func main() {
	fmt.Println(filepath.Join("http://example.com/", "/logout"))
	
}
---
http:/example.com/logout

Program exited.
```